gui/src/components/logs/Logs.svelte

```diff
@@ -18,15 +18,19 @@
       return;
     }
 
-    const threshold = 150;
+    const threshold = 20;
```

Comment:
Does this mean if you're scrolled more than 20 pixels up it won't snap new messages to the bottom?

Reply:
yeah, i should probably add a comment as to why 20 pixels...